packages/wonder-blocks-modal/components/modal-content.js

```diff
@@ -8,22 +8,40 @@ import ModalHeader from "./modal-header.js";
 
 type Props = {
     header?: React.Element<typeof ModalHeader> | React.Node,
+    scrollOverflow: boolean,
     children: React.Node,
     style?: any,
 };
 
 export default class ModalContent extends React.Component<Props> {
+    static defaultProps = {
+        scrollOverflow: true,
+    };
+
     render() {
-        const {header, style, children} = this.props;
+        const {header, scrollOverflow, style, children} = this.props;
 
         return (
-            <View style={styles.wrapper}>
+            <View
+                style={[
+                    styles.wrapper,
+                    scrollOverflow && styles.scrollOverflow,
+                ]}
+            >
                 {!header || header.type === ModalHeader ? (
                     header
                 ) : (
                     <ModalHeader>{header}</ModalHeader>
                 )}
-                <View style={[styles.content, style]}>{children}</View>
+                <View
+                    style={[
+                        styles.content,
+                        (gridSize) => gridSize === "small" && styles.small,
+                        style,
```

Comment:
I would usually expect styling to go on the root element of a component.  This breaks some implicit assumptions (that we should probably make explicit) about `style`.  In particular adding `margin` would add spacing within the scroll container not around it.  Perhaps we should rename `ModalContent`'s `style` prop to something else.

Reply:
Hmm - I'm not sure I follow. All the wrapper element is doing is adding the overflow - all the other styling is being applied to the element that's representing the contents. It definitely makes more sense to apply it to the contents (as being represented by what's passed in). Applying the user's styles to the wrapper would create some really strange effects (and I'm not sure that renaming it would improve things).

Reply:
In the docs we use `style` to add margin to add spacing between components.  This will lead people to think that you can always use `style` for that purpose.  In this case, the `style` prop on `ModalContent` can't be used for that purpose (afaict) so it should given a different name so that people don't get the wrong impression.

Reply:
Do you have a recommendation for the name? In your example, adding a custom marginTop to the content will move it farther away from the title bar/header, as intended. I can't think of a name that wouldn't just make it confusing (especially since we definitely don't want people adding styling to the outer wrapper).

Reply:
I guess this situation is a bit different b/c `ModalContent` will always be used inside `ModalPane`.  If it was used elsewhere then this would probably be a real problem but it sounds like the way it's currently set up it will meet dev expectations which is the important thing.